digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/AssetDataImporter.scala

```diff
@@ -1285,8 +1285,41 @@ def insertNumberPropertyData(propertyId: Long, assetId: Long, value:Int) {
       }else{
         sqlu"update text_property_value set value_fi = $vname, modified_date = sysdate, modified_by = $modifiedBy where asset_id = $assetId and property_id = $propertyVal".execute
       }
+    }
+
+  def insertNewAsset(typeId: Int, linkId: Long, startMeasure: Double, endMeasure: Double, sideCode: Int, value: Int) {
+    val assetId = Sequences.nextPrimaryKeySeqValue
+    val lrmPositionId = Sequences.nextLrmPositionPrimaryKeySeqValue
+    val propertyId = sql"""select id from property where public_id = 'mittarajoitus'""".as[Long].first
```

Comment:
There isn't any property associate for this new asset type Kaistojen lukumäärä(numberOfLanes), the frontend use this property_id to insert into the table "number_property_value". Note that the asset type related with this public_id is 30 (Kokonaispainorajoitukset)